pkg/api/vlabs/validate.go

```diff
@@ -1718,26 +1721,6 @@ func (a *Properties) validateCustomKubeComponent() error {
 	return nil
 }
 
-func (a *Properties) validatePrivateAzureRegistryServer() error {
```

Comment:
Is this removal related to the cloud-node-manager for Windows change?

Reply:
It's kinda related since I was using a cloud-node-manager image in my own ACR for testing. I don't think we should only be allowed to use `privateAzureRegistryServer` when `custom{Component}Image` is defined. That's why I removed this validation.